automation/test-execution/mlflow/stop-mlflow.sh

```
@@ -0,0 +1,85 @@
+    COMPOSE_CMD="docker compose"
+    RUNTIME="docker"
+elif command -v docker-compose &> /dev/null; then
+    COMPOSE_CMD="docker-compose"
+    RUNTIME="docker"
+else
+    COMPOSE_CMD=""
+    RUNTIME="docker"  # fallback
+fi
+
+# Stop by container name (compose)
+cd "$SCRIPT_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)